content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -31,9 +31,47 @@ Datadog CD Visibility integrates with Argo CD by using [Argo CD Notifications][2
 
 The setup below uses the [Webhook notification service][5] of Argo CD to send notifications to Datadog.
 
-First, add your [Datadog API Key][11] in the `argocd-notifications-secret` secret with the `dd-api-key` key. See [the Argo CD guide][2] for information on modifying the `argocd-notifications-secret`.
+First, add your [Datadog API Key][11] in the `argocd-notifications-secret` secret with the `dd-api-key` key. See [the Argo CD guide][2] for information on modifying the `argocd-notifications-secret`. For sending notifications, the setup is different depending on whether you installed Argo CD using Helm or the regular setup (ConfigMap).
```

Comment:
Given the regular setup is configmap, I think it should stay the first tab. This way most people will see it and not Helm